### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by BQTH** [katspaugh/wavesurfer.js#2696] [Q&A] [answered]
Hey, I was just wondering if there is a planning for releases, or if the next release is planned? Since a feature I build depends on an unreleased fix to function properly

**Comment by katspaugh** [comment]
AFAICT, these are the new commits since the last release:

* Fix typo (#2695)
* Fix iphone silent switch webaudio mute (#2686)
* Proxy HTMLMediaElement's 'waiting' event through MediaElement backend. (#2691)
* Docs: update the issue template and wavesurfer links (#2671)
* Create FUNDING.yml (#2668)
* regions plugin: improved delta calculation (resize end) (#2641)
* update dev dependencies
* doc: fix typo

(`git log --pretty=format:'* %s' 6.4.0...master`)

Looks like bug fixes mostly. I would say we could make a new release asap.

**Comment by katspaugh** [comment]
I've made a release PR here: https://github.com/wavesurfer-js/wavesurfer.js/pull/2705

**Comment by katspaugh** [comment, marked as answer]
6.5.0 is now live on NPM. 📦
